src/status_im/common/standard_authentication/standard_auth/slide_button/view.cljs

```diff
@@ -9,35 +9,33 @@
 
 (defn view
   [{:keys [track-text customization-color auth-button-label on-auth-success on-auth-fail
-           auth-button-icon-left size blur? container-style disabled? dependencies on-complete]
+           auth-button-icon-left size blur? container-style disabled? dependencies]
```

Comment:
I removed `on-complete` because we used it to prevent the standard authorization slider from doing standard authorization, which is a bit illogical :)
So now this slider works only for standard authorization, for cases when you need to choose between keycard or standard auth there is a simple slider.

Reply:
can you explain why the changes to standard_auth were needed in this case? Asking cause I'm finishing up on the PR which also removes it, but also allows to sign e.g transaction data with standard_auth (including the keycard support). So I'm wondering what was blocking it to make sure it's also supported there?

Reply:
update: changes were discussed on a call with @clauxx and we aligned our development.